server/package.json

```
@@ -1,6 +1,12 @@
 {
+  "author": "Han <admin@nyx-app.my.id>",
```

Biome (2.4.4)

[error] The key author was already declared.

(lint/suspicious/noDuplicateObjectKeys)